### PR TITLE
feature(sponsor edit page): apply new design

### LINKED ISSUE #695: Apply design edit page sponsor user
Dep: #566

### LINKED ISSUE #723: Apply design edit page sponsor(Mobile)

### PR DESCRIPTION
**Pull request by Edgar097**
Resolves #695
Resolves #723

### What does this PR do?
New sponsor edit page design

### How should this be tested?
- Make run
- Edit Sponsor

### CODE REVIEW

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]
@Edgar097 please, take a look at title space consistency, seem not to have same margin for each title

image

**Comment by leisterfrancisco** [comment]
@Edgar097 check merge conflicts

File: webapp/src/routes/EditProfile/EditProfileSponsorMobile.js [hunk 1]
```diff
@@ -0,0 +1,624 @@
...
+              width="100%"
+              style={{ display: showOrHideSocialMedia('instagram') }}
+            >
+              <SocialMediaTextField
+                textFieldClass={classes.mobileTextField}
+                idText="instagram-username"
+                name="instagram"
+                label={t('editProfile.instagramUsername')}
+                defaultValue={
+                  socialMedia.find((social) => social.name === 'instagram')
```
Comment (JustinCast) [inline webapp/src/routes/EditProfile/EditProfileSponsorMobile.js:464]: Same situation here

File: webapp/src/routes/EditProfile/EditProfileSponsorMobile.js [hunk 2]
```diff
@@ -0,0 +1,624 @@
...
+              width="100%"
+              style={{ display: showOrHideSocialMedia('facebook') }}
+            >
+              <SocialMediaTextField
+                textFieldClass={classes.mobileTextField}
+                idText="facebook-profile-url"
+                name="facebook"
+                label={t('editProfile.facebookProfileUrl')}
+                defaultValue={
+                  socialMedia.find((social) => social.name === 'facebook')
```
Comment (JustinCast) [inline webapp/src/routes/EditProfile/EditProfileSponsorMobile.js:488]: Same situation here

File: webapp/src/routes/EditProfile/EditProfileSponsorMobile.js [hunk 3]
```diff
@@ -0,0 +1,624 @@
...
+              width="100%"
+              style={{ display: showOrHideSocialMedia('twitter') }}
+            >
+              <SocialMediaTextField
+                textFieldClass={classes.mobileTextField}
+                idText="twitter-username"
+                name="twitter"
+                label={t('editProfile.twitterUsername')}
+                defaultValue={
+                  socialMedia.find((social) => social.name === 'twitter')
```
Comment (JustinCast) [inline webapp/src/routes/EditProfile/EditProfileSponsorMobile.js:511]: Same situation here

File: webapp/src/routes/EditProfile/EditProfileSponsorMobile.js [hunk 4]
```diff
@@ -0,0 +1,624 @@
...
+                  className={classes.cancelBtn}
+                >
+                  {t('common.cancel')}
+                </Button>
+              </Link>
+              <Button
+                className={classes.saveBtn}
+                variant="contained"
+                color="secondary"
+                onClick={() => prepareDataForSubmitting()}
```
Comment (JustinCast) [inline webapp/src/routes/EditProfile/EditProfileSponsorMobile.js:604]: `onClick={prepareDataForSubmitting}`

File: webapp/src/routes/EditProfile/styles.js [hunk 5]
```diff
@@ -239,9 +259,31 @@ export default (theme) => ({
...
+  textFieldSocialMedia: {
+    height: 'auto',
+    overflowY: 'none',
+  },
+  mapField: {
+    height: 'auto',
+    overflowY: 'none',
+    margin: theme.spacing(2, 0),
+    width: '100%',
+    height: '287px'
```
Comment (JustinCast) [inline webapp/src/routes/EditProfile/styles.js:280]: Duplicate key height

**Review by JustinCast — APPROVED** [review]
👍

**Review by steph222 — APPROVED** [review]

**Pull request merged**